Update hybrid_planning_tutorial.rst , launchfile needs to import plugin differently

marc-wittwer opened the pull request:
### Description

The classes cannot be found if the plugin import was written like in the tutorial. 

```
[ERROR] [launch_ros.actions.load_composable_nodes]: Failed to load node 'global_planner' of type 'moveit_hybrid_planning::GlobalPlannerComponent' in container '/hybrid_planning_container': Failed to find class with the requested plugin name.
[component_container-5] [ERROR] [1661345481.458624976] [hybrid_planning_container]: Failed to find class with the requested plugin name 'moveit_hybrid_planning::GlobalPlannerComponent' in the loaded library
[component_container-5] [ERROR] [1661345481.470274994] [hybrid_planning_container]: Failed to find class with the requested plugin name 'moveit_hybrid_planning::LocalPlannerComponent' in the loaded library
[ERROR] [launch_ros.actions.load_composable_nodes]: Failed to load node 'local_planner' of type 'moveit_hybrid_planning::LocalPlannerComponent' in container '/hybrid_planning_container': Failed to find class with the requested plugin name.
[ERROR] [launch_ros.actions.load_composable_nodes]: Failed to load node 'hybrid_planning_manager' of type 'moveit_hybrid_planning::HybridPlanningManager' in container '/hybrid_planning_container': Failed to find class with the requested plugin name.
[component_container-5] [ERROR] [1661345481.480689611] [hybrid_planning_container]: Failed to find class with the requested plugin name 'moveit_hybrid_planning::HybridPlanningManager' in the loaded library
```

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

JafarAbdi commented:
Hi @marc-wittwer, I can't push to your branch, do you mind rebasing on the main branch? I add labels for mergify to automatically cherry-pick this PR for the other branches!

Thanks!

marc-wittwer commented:
@JafarAbdi I added you as a collaborator to my fork. 
I tried to rebase, however 18 merge conflicts showed up which left me quite overwhelmed. 
Thus, I did not manage to rebase sorry.

```
git rebase main patch-1
warning: skipped previously applied commit b5dbec431
warning: skipped previously applied commit e045ea360
warning: skipped previously applied commit 97dea6df7
warning: skipped previously applied commit 214be4d3e
warning: skipped previously applied commit bac3d9417
hint: use --reapply-cherry-picks to include skipped commits
hint: Disable this message with "git config advice.skippedCherryPicks false"
dropping 39430fdd02b89020eee51b3862064e519ab6dbc5 Fix broken link to controller config file (#351) -- patch contents already upstream
Auto-merging doc/tutorials/getting_started/getting_started.rst
CONFLICT (content): Merge conflict in doc/tutorials/getting_started/getting_started.rst
error: could not apply fab1de904... Add branch info to Galactic build (#352)
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply fab1de904... Add branch info to Galactic build (#352)
marc@omen:~/Downloads/moveit2_tutorials (patch-1|REBASE 4/18)$ code .
marc@omen:~/Downloads/moveit2_tutorials (patch-1|REBASE 4/18)$ git status
interactive rebase in progress; onto 4b960b2db
Last commands done (4 commands done):
   pick 39430fdd0 Fix broken link to controller config file (#351)
   pick fab1de904 Add branch info to Galactic build (#352)
  (see more in file .git/rebase-merge/done)
Next commands to do (14 remaining commands):
   pick d50d9a56f Change moveit_visual_tools version to 4.0.1, remove depreciated constructor of PlanningSceneMonitor (#361)
   pick a01f47e52 Enable Humble Tutorial (#373) (#375)
  (use "git rebase --edit-todo" to view and edit)
You are currently rebasing branch 'patch-1' on '4b960b2db'.
  (all conflicts fixed: run "git rebase --continue")

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   doc/tutorials/getting_started/getting_started.rst

marc@omen:~/Downloads/moveit2_tutorials (patch-1|REBASE 4/18)$ git status
interactive rebase in progress; onto 4b960b2db
Last commands done (5 commands done):
   pick fab1de904 Add branch info to Galactic build (#352)
   pick d50d9a56f Change moveit_visual_tools version to 4.0.1, remove depreciated constructor of PlanningSceneMonitor (#361)
  (see more in file .git/rebase-merge/done)
Next commands to do (13 remaining commands):
   pick a01f47e52 Enable Humble Tutorial (#373) (#375)
   pick 245554883 Make humble default page (#378) (#383)
  (use "git rebase --edit-todo" to view and edit)
You are currently rebasing branch 'patch-1' on '4b960b2db'.
  (fix conflicts and then run "git rebase --continue")
  (use "git rebase --skip" to skip this patch)
  (use "git rebase --abort" to check out the original branch)

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   doc/examples/realtime_servo/src/servo_cpp_interface_demo.cpp

Unmerged paths:
  (use "git restore --staged <file>..." to unstage)
  (use "git add <file>..." to mark resolution)
	both modified:   moveit2_tutorials.repos

```
